tests/reporting/formatting/test_resources/markdown_run.md

```diff
@@ -6,5 +6,5 @@ Build failed
 ```
 🏗️  _dockertest_, _test_  
 📋  _test_  
-🧪 51 ❌ 1 💔 0 🙈 0 [link](http://localhost/tests) 
+🧪 51 ❌ 1 💔 0 🙈 0 [Docker Test](http://localhost/tests)  
```

Comment:
The extra spaces are on purpose, it fixes an issue where locally the lines after the results weren't on a newline. The other lines also have 2 spaces